randovania/games/fusion/logic_database/Sector 4 AQA.txt

```diff
@@ -1031,7 +1042,8 @@ Extra - room_id: [24]
       All of the following:
           Missiles
           Any of the following:
-              Can Use Any Bombs
+              Can Use Power Bombs
+              Hi-Jump and Can Use Bombs
```

Comment:
Why is HJ needed? Are you sure you didn't mean the spring ball template instead?